osu.Game/Screens/Play/PlayerSettings/InputSettings.cs

```diff
@@ -23,9 +23,8 @@ private void load(OsuConfigManager config)
             {
                 new PlayerCheckbox
                 {
-                    // TODO: change to touchscreen detection once https://github.com/ppy/osu/pull/25348 makes it in
```

Comment:
I felt like this change made sense, but I guess it's not a big deal either way for now.